cmd/root.go

```diff
@@ -68,13 +76,89 @@ func requiredStringFlags(names ...string) error {
 
 func preRunEOutput(cmd *cobra.Command, args []string) error {
 	// Initializes/Validates the HCL and TFSTATE flags
-	if viper.GetString("hcl") != "" {
-		f, err := os.OpenFile(viper.GetString("hcl"), os.O_APPEND|os.O_TRUNC|os.O_RDWR|os.O_CREATE, 0644)
+	if viper.GetString("module") != "" {
+		module := viper.GetString("module")
```

Comment:
This could be (I see you did that on another file, haha)

```go
if module := viper.GetString("module"); module != "" {
...
```

But i'd say ignore it, not worth another RS 😆